src/main/java/io/cdap/plugin/gcp/gcs/StorageClient.java

```diff
@@ -45,6 +45,10 @@
   private static final Logger LOG = LoggerFactory.getLogger(StorageClient.class);
   private final Storage storage;
 
+  public Storage getStorage() {
```

Comment:
should not have this. Instead, make the constructor public and make the caller pass in the Storage class and use it outside of this class.

Reply:
I will not need this because will be moving that part of code to StorageClient class itself.